Cargo.toml

```diff
@@ -11,20 +11,20 @@ license = "MIT"
 name = "bevy_normal_material"
 readme = "README.md"
 repository = "https://github.com/mattatz/bevy_normal_material"
-version = "0.6.0"
+version = "0.7.0"
 
 [dependencies.bevy]
 default-features = false
 features = ["bevy_render", "bevy_pbr", "bevy_asset", "tonemapping_luts"]
-version = "0.14.2"
+version = "0.15.0"
 
 [dev-dependencies.bevy]
 default-features = false
 features = ["bevy_render", "bevy_pbr", "bevy_core_pipeline", "bevy_asset"]
-version = "0.14.2"
+version = "0.15.0"
 
 [features]
-examples = ["bevy/bevy_render", "bevy/bevy_pbr", "bevy/bevy_core_pipeline", "bevy/bevy_winit", "bevy/x11"]
+examples = ["bevy/bevy_render", "bevy/bevy_pbr", "bevy/bevy_core_pipeline", "bevy/bevy_winit", "bevy/x11", "bevy/bevy_window"]
```

Comment:
Without "bevy/bevy_window" I get a panic when trying to run the examples using Bevy 0.15.

Resource requested by (bevy_ecs::system::commands::Commands, bevy_ecs::system::query::Query<(bevy_ecs::entity::Entity, &mut bevy_window::window::Window, core::option::Option<&bevy_window::raw_handle::RawHandleWrapperHolder>), bevy_ecs::query::filter::Added<bevy_window::window::Window>>, bevy_ecs::event::writer::EventWriter<bevy_window::event::WindowCreated>, bevy_ecs::change_detection::NonSendMut<bevy_winit::winit_windows::WinitWindows>, bevy_ecs::change_detection::NonSendMut<bevy_winit::accessibility::AccessKitAdapters>, bevy_ecs::change_detection::ResMut<bevy_winit::accessibility::WinitActionRequestHandlers>, bevy_ecs::change_detection::Res<bevy_a11y::AccessibilityRequested>, bevy_ecs::change_detection::Res<bevy_winit::winit_monitors::WinitMonitors>) does not exist: bevy_ecs::event::collections::Events<bevy_window::event::WindowCreated